playground.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+""" Playground class for the random walk simulation """
+
+from shapely.geometry import Polygon, Point
+
+
+class Playground:
+    """Defines a playground with obstacles for RandomWalkers"""
+    def __init__(self, scaling: int = 1, x_max: int = 250, y_max: int = 250, seed: int = 0):
```

Comment:
the fix size of the playground could be changed so that it adapts to the number of walkers -> would make the plot clearer as well

Reply:
set default sacling to 4 in the main function

Reply:
adapt scaling to number of walkers would not change anything.